Index: app/src/main/java/com/comp3350_group10/bookstore/persistence/IBookDatabase.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>/**\n * Interface for BookDatabase\n */\n\npackage com.comp3350_group10.bookstore.persistence;\n\nimport java.util.List;\n\npublic interface IBookDatabase\n{\n    List<IBook> findBook(String searchTerm) throws ClassNotFoundException;\n    List<IBook> getBooks() throws ClassNotFoundException;\n    void insertBook(IBook book) throws ClassNotFoundException;\n    void updateBook(IBook book) throws ClassNotFoundException;\n    void deleteBook(IBook book) throws ClassNotFoundException;\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/comp3350_group10/bookstore/persistence/IBookDatabase.java	(revision 51b158d7e80350252c423a1f37c7284a785e784e)
+++ app/src/main/java/com/comp3350_group10/bookstore/persistence/IBookDatabase.java	(date 1617238953000)
@@ -10,7 +10,7 @@
 {
     List<IBook> findBook(String searchTerm) throws ClassNotFoundException;
     List<IBook> getBooks() throws ClassNotFoundException;
-    void insertBook(IBook book) throws ClassNotFoundException;
-    void updateBook(IBook book) throws ClassNotFoundException;
+    IBook insertBook(IBook book) throws ClassNotFoundException;
+    IBook updateBook(IBook book) throws ClassNotFoundException;
     void deleteBook(IBook book) throws ClassNotFoundException;
 }
